### DISCUSSION TITLE
v1.9.0: Inactive reminders

### DISCUSSION THREAD

**Discussion by Futsch1** [Futsch1/medTimer#218] [Announcements]
Release v1.9.0 saw an increase of the minor version because the database layout was changed. This was necessary because now a reminder can be deactivated in their advanced setting. In this state, they do not trigger any reminders. However, they are now available as additional options in the "Log additional dose" selection, where their amount is now directly used and makes adding a reoccurring additional dose with a specific amount easier.

<img src='https://github.com/user-attachments/assets/9a9f1555-0fdc-4f63-988d-3ad22068d5a9' width=150/>

**Comment by VJ349** [comment]
Good feature to deactivate meds. Would you consider displaying "Inactive" in the Medicine view against medicines that have been marked that way? Perhaps instead of the "no reminders set" text?

**Reply by Futsch1** [reply]
That's what I already implemented for the upcoming release 👍

See issue #227

**Reply by VJ349** [reply]
👍
